packages/celotool/src/e2e-tests/governance_tests.ts

```diff
@@ -694,6 +718,37 @@ describe('governance tests', () => {
     })
   })
 
+  describe('when rewards distribution is frozen', () => {
+    before(restart)
+
+    let epoch: number
+    let blockFrozen: number
+    let latestBlock: number
+
+    beforeEach(async function(this: any) {
```

Comment:
I would think this should be a `before` instead of `beforeEach` because you only need to do it once. Or am I mistaken?